### PR TITLE
[es] Add Glossary entry for feature gate

### PR DESCRIPTION
**Pull request by ManriqueGarcia**

### CODE REVIEW

**Comment by NitishKumar06** [comment]
HI @ManriqueGarcia ! Please sign the CLA.

File: content/es/docs/reference/glossary/feature-gate.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Feature Gate
+id: feature-gate
+date: 2022-06-07
+full_link: /docs/reference/command-line-tools-reference/feature-gates/
+short_description: >
+  Las feature gates, puertas de funciones en castellano, son un conjunto de pares clave=valor que  permiten habilitar o deshabilitar al usuario funciones experimentales en Kubernetes.
```
Comment (electrocucaracha) [inline content/es/docs/reference/glossary/feature-gate.md:7]: Me parece que está de más la traducción del término
```suggestion
  Las feature gates son un conjunto de pares clave-valor que permiten al usuario habilitar o deshabilitar funciones experimentales en Kubernetes.
```

File: content/es/docs/reference/glossary/feature-gate.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+aka:
+tags:
+- component-tools
+---
+Las feature gates, puertas de funciones en castellano, son un conjunto de pares clave=valor que  permiten habilitar o deshabilitar     al usuario funciones experimentales en Kubernetes.
```
Comment (electrocucaracha) [inline content/es/docs/reference/glossary/feature-gate.md:13]:
```suggestion
Las feature gates son un conjunto de pares clave-valor que permiten al usuario habilitar o deshabilitar funciones experimentales en Kubernetes.
```

File: content/es/docs/reference/glossary/feature-gate.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+<!--more-->
+Las puertas de funciones están destinadas a cubrir el ciclo de vida de desarrollo de una función; no están destinadas a ser API a largo plazo. Como tal, se espera que queden en desuso y se eliminen después de que una característica se convierta en GA o se elimine.
```
Comment (electrocucaracha) [inline content/es/docs/reference/glossary/feature-gate.md:16]:
```suggestion
Las feature gates están destinadas a cubrir el ciclo de vida de desarrollo de una función; no están destinadas a ser APIs a largo plazo. Como tal, se espera que queden en desuso y se eliminen después de que una característica se convierta en GA o se elimine.
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]
As @NitishKumar06 mentioned before, you have to sign the CLA. Let me know if you have questions about that process.

**Comment by Sea-n** [comment]
You need to add `re***ci@redhat.com` to [GitHub Setting > Email](https://github.com/settings/emails) and sign [the CLA agreement](https://github.com/kubernetes/website/pull/34214#issuecomment-1152291412).
Please let us know if there are any issue.

**Review by Sea-n — CHANGES_REQUESTED** [review]
Where did this translation come from?

I don't see `content/en/docs/reference/glossary/feature-gate.md` file in English.

And in your `full_link` metadata, the [cli-ref/feature-gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is totally different with this.

/hold
/retitle [es] Add Glossary feature gate

**Comment by electrocucaracha** [comment]
> Where did this translation come from?
> 
> I don't see `content/en/docs/reference/glossary/feature-gate.md` file in English.
> 
> And in your `full_link` metadata, the [cli-ref/feature-gates](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) is totally different with this.
> 
> /hold /retitle [es] Add Glossary feature gate

I saw the [_index.md file](https://github.com/kubernetes/website/blob/main/content/es/docs/reference/command-line-tools-reference/_index.md) has an extra `toc-hide: true` line which maybe is affecting the rendering. @ManriqueGarcia can you remove that extra line.

**Comment by ManriqueGarcia** [comment]
> toc-hide: true

I only see one line with `toc-hide:true` in the file should the whole line be removed?

**Comment by sftim** [comment]
/retitle [es] Add Glossary entry for feature gate

/hold
This is not localizing an existing English file, and for that reason we should not merge it.

**Comment by ManriqueGarcia** [comment]
> /retitle [es] Add Glossary entry for feature gate
> 
> /hold This is not localizing an existing English file, and for that reason we should not merge it.

so, do I create the entry also in English?

**Comment by Sea-n** [comment]
Hey @ManriqueGarcia,

Thanks for purposing this PR, but our policy doesn't want pages diverged in each language.

If you think this page is good (and so do I), please open a new PR to add this page in English first.

This intention is really good, but I'm afraid that I have to close this.
/close

**Pull request closed without merging**